src/azul/service/responseobjects/storage_service.py

```diff
@@ -36,6 +37,15 @@ def put(self, object_key: str, data: bytes, content_type: Optional[str] = None,
 
         return object_key
 
+    def upload(self, file_name, object_key):
+        aws_profile = os.getenv('AWS_PROFILE')
+        session = boto3.Session(profile_name=aws_profile)
+        s3 = session.resource('s3')
+        s3.meta.client.upload_file(Filename=file_name,
```

Comment:
Why arent't you using self.client()? Even if `self.client()` wasn't available, this would still be a needlessly complicated way to get to a boto3 client.

Reply:
fixed